pyscriptjs/src/pyodide.ts

```diff
@@ -54,8 +58,8 @@ export class PyodideRuntime extends Runtime {
     async loadInterpreter(): Promise<void> {
         logger.info('Loading pyodide');
         this.interpreter = await loadPyodide({
-            stdout: console.log,
-            stderr: console.log,
+            stdout: (msg) => { this.stdio.stdout_writeline(msg); },
```

Comment:
Do we need to add typing here or does TS knows `msg` will be string?

Reply:
Uhm, good question, I don't really know.
I tried to compile with `--strict` and the compiler doesn't give any warning, so maybe somehow it manages to infer it. But adding an explicit type doesn't hurt, so I added it in 095f156